schema-supplement.rst

```diff
@@ -90,6 +90,7 @@ By definition, none of the following attributes are required.
 .. cps:attribute:: website
   :type: string
   :context: package
+  :format: uri
```

Comment:
Oops, that's because of a dumb typo; the Python code uses the variable name `typeformat` because `format` is a built-in, but accidentally used that also as the name of the reST option to inspect. Should be fixed; thanks for the catch!